lib/fetch/index.js

```diff
@@ -1867,7 +1867,9 @@ async function httpNetworkFetch (
 
       // 7. Enqueue a Uint8Array wrapping an ArrayBuffer containing bytes
       // into stream.
-      fetchParams.controller.controller.enqueue(new Uint8Array(bytes))
+      if (!isAborted(fetchParams)) {
+        fetchParams.controller.controller.enqueue(new Uint8Array(bytes))
+      }
```

Comment:
Is this a spec bug or are we doing something wrong somewhere else?

Reply:
Possibly a spec bug or way undici handles it. When a request is aborted, the fetchparams.controller.resume can still run (causing the bug). It could also be step 18 of https://fetch.spec.whatwg.org/#http-network-fetch but I think "if aborted" refers to *previous* steps (although I'm still confused by it).

Reply:
@KhafraDev can you move this to after await next(). That's the part where the check is relevant since we go async there.